versions/3.1.0.md

```diff
@@ -2336,6 +2336,20 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+OpenAPI documents are made up of multiple resources, sometimes resources are all in the same root OpenAPI document (inline resources), or resources are contain their own documents with `$ref` (separate resources). A schema object could exist as an inline resource, or a separate resource. 
```

Comment:
```suggestion
OpenAPI documents are made up of multiple resources. Sometimes resources are all in the same root OpenAPI document (inline resources); also, resources cam contain their own documents with `$ref` (separate resources). A schema object could exist as an inline resource, or a separate resource. 
```

Reply:
@karenetheridge `cam` => `can`

Reply:
doh, fixed

Reply:
Where words like "could" and "can" take the meanings of words in the [RFC2119](https://tools.ietf.org/html/rfc2119) list like "MAY", we usually prefer those (and they should be capitalised).